inference-engine/src/vpu/graph_transformer/src/middleend/allocator/allocator.cpp

```diff
@@ -77,8 +77,6 @@ void updateChildDataAllocation(const Data& data, int offsetLimitation) {
             }
 
             memoryOffset += byteOffset;
-
-            IE_ASSERT(memoryOffset + child->lastElemOffset() <= offsetLimitation);
```

Comment:
You are removing an assert in CMX case
You can pass `_maxCmxSize` and do something like
`IE_ASSERT(parent->dataLocation() != Location::CMX || memoryOffset + child->lastElemOffset() <= offsetLimitation);`